app/scripts/comp/launcher-electron.js

```diff
@@ -0,0 +1,266 @@
+'use strict';
```

Comment:
yes, I did in latest commits: 
https://github.com/keeweb/keeweb/pull/505/commits/db701754b0c4e5fff19dd92c69fef8563c96d906